static/css/impala/hovercards.less

```diff
@@ -83,7 +83,7 @@
         margin-top: 8px;
         font-size: 11px;
         font-style: italic;
-
+        max-width: 220px;
```

Comment:
This is a nitpick, but could you move this up with the other styles and keep the newline to separate these styles, eg:

```less
.byline {
    font-size: 11px;
    font-style: italic;
    margin-top: 8px;
    max-width: 220px;

    a {
        [...]
```